lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -196,8 +204,11 @@ abstract class _BaseAdapter {
       }
     };
     _listeners[eventName] = loggedHandler;
-    glassPaneElement
-        .addEventListener(eventName, loggedHandler, true);
+    // We have to attach the event listener on the window instead of the
+    // glasspane element. That's because "up" events that occur outside the
+    // browser are only reported on window, not on DOM elements.
+    // See: https://github.com/flutter/flutter/issues/52827
+    html.window.addEventListener(eventName, loggedHandler, true);
```

Comment:
have you verified this on Firefox/safari as well ?

Reply:
Good point. I'll verify.

Reply:
There has been back'n'forth about global event attachment. Can you please check the history to make sure this is not regressing some corner cases, such as a11y, platform views, iframes?

Reply:
Tested in Firefox and Safari. Works like a charm!